enclave/core/backtrace.c

```diff
@@ -33,7 +33,7 @@ const void* _check_address(const void* ptr)
  * This new implementation below safely walks up the call-stack, ensuring that
  * each potential-frame is not null and lies within the enclave.
  */
-int oe_backtrace(void** buffer, int size)
+size_t oe_backtrace(void** buffer, size_t size)
```

Comment:
Retain the original signature for oe_backtrace and oe_bactrace_symbols.
These are based on gnu backtrace:  https://www.gnu.org/software/libc/manual/html_node/Backtraces.html